tutorial/examples/notebooks/xarray2pandas.ipynb

```diff
@@ -324,7 +324,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python [conda env:notebook] *",
+   "display_name": ""Python 3",
    "language": "python",
    "name": "conda-env-notebook-py"
```

Comment:
```suggestion
   "name": "python3"
```
and you will have to do that same (change `display_name` and `name`) in:
- `tutorial/examples/notebooks/xesmf_regridding.ipynb`
- `tutorial/pangeo101/dask_introduction.ipynb`

better would probably be a `pre-commit` hook that would automatically change this for you... in `xarray-tutorial` we put this in [`.pre-commit-config.yaml`](https://github.com/xarray-contrib/xarray-tutorial/blob/main/.pre-commit-config.yaml), among others:
```yaml
  - repo: https://github.com/kynan/nbstripout
    rev: 0.6.1
    hooks:
      - id: nbstripout
        args: [--extra-keys=metadata.kernelspec metadata.language_info.version]
```
and a bot will run that on each push to a PR (that's a bit too much for today, though).